### DISCUSSION TITLE
User script

### DISCUSSION THREAD

**Discussion by katastrophe92** [ferdium/ferdium-app#1162] [Q&A] [answered]
Hey y'all. If Ferdium doesn't have extensions, it is compatible with User Scripts?

**Comment by marie-bnl** [comment, marked as answer]
Yes it is. Right click on the services, then Edit and “Open user.js”.

**Comment by katastrophe92** [comment]
hey @victorbnl . What type of scripts work or what are the limitations? Thanks

**Reply by marie-bnl** [reply]
I don’t know, I don’t use it myself. Try it and see!
